### PR TITLE
add some documents and minor codestyle change

### PR DESCRIPTION
**Pull request by trungams**
- Create contributing.md and pull_request_template.md
- Remove duplicate imports of `asyncio` in Main.py
- Move log file pointer to instance attribute of memes object

### CODE REVIEW

File: cogs/memes.py [hunk 1]
```diff
@@ -11,6 +11,10 @@
 class Memes:
     def __init__(self, bot):
         self.bot = bot
+        self.mix_command_logger = open('mix.log', 'a')
+
+    async def on_disconnect(self):
+        self.mix_command_logger.close()
```
Comment (idodin) [reply cogs/memes.py]: If the python process running the script crashes for some reason, there's no guarantee that it'll be run. I think discord-py will run it before exiting in the case of a runtime error though...

Comment (trungams) [reply cogs/memes.py]: @davidlougheed it's here: https://discordpy.readthedocs.io/en/rewrite/api.html?highlight=on_disconnect#discord.on_disconnect

IIRC the discord.py client has a handler for bot closing event

Comment (davidlougheed) [inline cogs/memes.py]: I can't find info on this in the docs; do you know if this is guaranteed to be called even if there's a crash?

Comment (davidlougheed) [reply cogs/memes.py]: ok, we may want to manually flush the log occasionally then, but not a big deal.

Comment (idodin) [reply cogs/memes.py]: Just as a note, IIRC from my other bot, discord completes disconnect on runtime error so the on_disconnect should still be called.

**Pull request merged**